networkit/cpp/numerics/LAMG/MultiLevelSetup.cpp

```diff
@@ -87,7 +87,7 @@ void MultiLevelSetup<CSRMatrix>::computeAffinityMatrix(const CSRMatrix& matrix,
     for (omp_index i = 0; i < static_cast<omp_index>(matrix.numberOfRows()); ++i) {
         double nir = 1.0 / normSquared[i];
         index cIdx = rowIdx[i];
-        matrix.forNonZeroElementsInRow(i, [&](index j, double val) {
+        matrix.forNonZeroElementsInRow(i, [&](index j, double) {
```

Comment:
Only minor:In the header-file there are also functions with anonymous parameters, but their styling is slightly different like `double /* val */`. Better use one or the other.